apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.ts

```diff
@@ -310,6 +314,12 @@ export class PurchaseRegisterExpandComponent implements OnInit, OnDestroy {
                 "checked": true,
             },
         ];
+
+        this.store.pipe(select(state => state.session.activeCompany), takeUntil(this.destroyed$)).subscribe(activeCompany => {
```

Comment:
use component store

Reply:
Sir, here no component store is implemented that's why I use global store.